src/Parallel/spanningtrees/boruvka-multithread.jl

```diff
@@ -0,0 +1,123 @@
+using LightGraphs, SimpleWeightedGraphs
+using CPUTime
+using DelimitedFiles
+using Base
+using DataStructures
+using Base.Threads
+
+"""
+    boruvka_mst_multithread(g)
+
+Return a tuple `(mst, weights)` where `mst` is a vector of edges representing the
+optimum (minimum, by default) spanning tree of a connected, undirected graph
+`g` , and `weights` is the sum of all the edges in the solution by using
+[Boruvka's algorithm](https://en.wikipedia.org/wiki/Bor%C5%AFvka%27s_algorithm).
+The algorithm requires that all edges have different weights to correctly generate a minimun/maximum spanning tree
+### Optional parameter(s):
+`maxItr`: Used to limit maximum number of iterations the algorithm will be performed. The default is log2(numVertex).
+"""
+function boruvka_mst_multithread(g::SimpleWeightedGraph, maxItr = convert(Int64, round(log2(nv(g))+1, digits=0)))
```

Comment:
`maxIter = round(Int64, log2(nv(g)) + 1)`

Reply:
Oh so I do not need to convert it to Integer?